server/src/server.js

```diff
@@ -70,7 +72,7 @@ wss.on('connection', async (socket, request) => {
 
   socket.once('close', () => {
     console.log('socket::close [sessionId:%s]', socket.sessionId);
-    
```

Comment:
unneeded space.

Reply:
Origional code has extra space at the end of the line, my editor auto trim them